audiencia/boy/config.txt
#
# HCApplet config file, all of these name value pairs can be placed in 
# PARAM tags in the APPLET tag if desired
#
################################################################

HCDataSetFile_version
"1.0"
HCDataSetFile_fields
"0"			"name"
"1"			"value"

HCDataSetFile_data
"id"			"chaticons"
"name"			"StorTrooper Designer"
"author"		"Stor Entertainment Ltd"
"authorEmail"		"goods@stor.co.uk"
"infoUrl"		"http://www.stortroopers.com/info.html"
"infoTarget"		"_blank"
"helpUrl"		"http://www.stortroopers.com/help.html"
"helpTarget"		"main"
"size"			"exact"
"inline"		"yes"
"popout"		"no"
"startsAs"		"inline"

"loadingText"		" The storTrooper editor is loading, complete with base wardrobe. It could take a little while, so hang in there...!"
"loadingBGColor"	"666600"
"loadingTextColor"	"ffffff"
"loadingFontSize"	"12"
"loadingFontFace"	"georgia"
"loadingTextX"		"15"
"loadingTextY"		"45"
"defaultTarget"		"main"
"mainBGColor"		"ffffff"
"bottomBGColor"		"999999"
"articlesBGColor"	"ffffff"
"dataPath"		"data/"
"imagePath"		"data/images/"
"articlesImagePath"	"data/images/articles/"

"gifAndArticlesPostURL" "save.php"
"saveArticlePostURL"	"admin_save.php"

"scrollbarBgColor"      "ffffff"
"scrollbarColor"        "666600"
"scrollbarBorderColor"  "000000"


# Relative to dataPath
"articlesConfigFile"	"articles.php"

# url and target of frame containing users icon, so it can be refreshed
"pullUrl"		"http://www.stortroopers.com/classic_boy"
"pullTarget"		"_self"

# layer order definitions-currently unused type should be set to "unspecified"
"layer0"  "unspecified"
"layer1"  "behind"
"layer2"  "unspecified"
"layer3"  "body"

"layer4"  "unspecified"
"layer5"  "hair"
"layer6"  "unspecified"
"layer7"  "underware"
"layer8"  "unspecified"
"layer9"  "tops n bottoms"
"layer10"  "unspecified"
"layer11"  "shoes"
"layer12"  "unspecified"
"layer13"  "jackets"
"layer14"  "hats"
"layer15"  "infront"
"layer16"  "unspecified"



# By setting "noLimits_"+layername to "yes" means that you do not have to 
# remove the above layers of clothes before changing this layer (e.g. would
# be silly to have to remove a jacket to have to change the shoes, or 
# visa-versa) Also it means that this layer is not only not effected by
# other clothes layers, it's does not affect other clothes layers. This 
# means that where it is in the layer stack only effects the plotting
# of the article!
"noLimits_infront"   "yes"
"noLimits_hats"	     "yes"
"noLimits_shoes"     "yes"
"noLimits_hair"	     "yes"
"oneItem_body"	     "yes"



audiencia/girl/config.txt
#
# HCApplet config file, all of these name value pairs can be placed in 
# PARAM tags in the APPLET tag if desired
# 
################################################################

HCDataSetFile_version
"1.0"

HCDataSetFile_fields
"0"			"name"
"1"			"value"


HCDataSetFile_data
"id"			"chaticons"
"name"			"StorTrooper Designer"
"author"		"Stor Entertainment Ltd"
"authorEmail"		"goods@stor.co.uk"
"infoUrl"		"http://www.stortroopers.com/info.html"
"infoTarget"		"_blank"
"helpUrl"		"http://www.stortroopers.com/help.html"
"helpTarget"		"main"
"size"			"exact"
"inline"		"yes"
"popout"		"no"
"startsAs"		"inline"

"loadingText"		" The storTrooper editor is loading, complete with base wardrobe. It could take a little while, so hang in there...!"
"loadingBGColor"	"666600"
"loadingTextColor"	"ffffff"
"loadingFontSize"	"12"
"loadingFontFace"	"georgia"
"loadingTextX"		"15"
"loadingTextY"		"45"
"defaultTarget"		"main"
"mainBGColor"		"ffffff"
"bottomBGColor"		"999999"
"articlesBGColor"	"ffffff"
"dataPath"		"data/"
"imagePath"		"data/images/"
"articlesImagePath"	"data/images/articles/"

"gifAndArticlesPostURL" "save.php"
"saveArticlePostURL"	"admin_save.php"

"scrollbarBgColor"      "ffffff"
"scrollbarColor"        "666600"
"scrollbarBorderColor"  "000000"


# Relative to dataPath
"articlesConfigFile"	"articles.php"


# url and target of frame containing users icon, so it can be refreshed

"pullUrl"		"http://www.stortroopers.com/classic_girl"
"pullTarget"		"_self"


# layer order definitions-currently unused type should be set to "unspecified"
"layer0"  "unspecified"
"layer1"  "behind"
"layer2"  "unspecified"
"layer3"  "body"
"layer4"  "unspecified"
"layer5"  "hair"
"layer6"  "unspecified"
"layer7"  "underware"
"layer8"  "unspecified"
"layer9"  "tops n bottoms"
"layer10"  "unspecified"
"layer11"  "shoes"
"layer12"  "unspecified"
"layer13"  "jackets"
"layer14"  "hats"
"layer15"  "infront"
"layer16"  "unspecified"


# By setting "noLimits_"+layername to "yes" means that you do not have to 
# remove the above layers of clothes before changing this layer (e.g. would
# be silly to have to remove a jacket to have to change the shoes, or 
# visa-versa) Also it means that this layer is not only not effected by 
# other clothes layers, it's does not affect other clothes layers. This 
# means that where it is in the layer stack only effects the plotting
# of the article!
"noLimits_infront"   "yes"
"noLimits_hats"	     "yes"
"noLimits_shoes"     "yes"
"noLimits_hair"	     "yes"
"oneItem_body"	     "yes"





audiencia/fashion_girl/config.py
import os
#"0"	"id"
#"1"	"imageName"
#"2"	"category"
#"3"	"layer"
#"4"	"snapPosX"
#"5"	"snapPosY"
#"6" "wearing"
#"7" "waldrobe"

items = ID, ImageName, Category, Layer, SnapPosX, SnapPosY, Wearing, Waldrobe \
    = range(8)

#from wget http://www.stortroopers.com/classic_boy/data/articles.php
configdata = """ "1" "girl_body_01.gif" "body" "body" "27" "28" "-1"
"2" "girl_body_02.gif" "body" "body" "27" "28" "-1"
"3" "girl_body_03.gif" "body" "body" "27" "28" "-1"
"4" "girl_body_04.gif" "body" "body" "27" "28" "-1"
"5" "alien.gif" "body" "body" "27" "28" "-1"
"6" "girl_skirt_01.gif" "bottoms" "tops n bottoms" "36" "96" "-1"
"7" "girl_skirt_02.gif" "bottoms" "tops n bottoms" "36" "94" "-1"
"8" "girl_skirt_03.gif" "bottoms" "tops n bottoms" "33" "91" "-1"
"9" "girl_skirt_04.gif" "bottoms" "tops n bottoms" "39" "96" "-1"
"10" "girl_skirt_05.gif" "bottoms" "tops n bottoms" "39" "96" "-1"
"11" "girl_skirt_06.gif" "bottoms" "tops n bottoms" "38" "98" "-1"
"12" "girl_skirt_07.gif" "bottoms" "tops n bottoms" "38" "98" "-1"
"13" "girl_jeans_01.gif" "bottoms" "tops n bottoms" "35" "98" "-1"
"14" "girl_jeans_02.gif" "bottoms" "tops n bottoms" "36" "97" "-1"
"15" "girl_jeans_03.gif" "bottoms" "tops n bottoms" "38" "98" "-1"
"16" "girl_jeans_04.gif" "bottoms" "tops n bottoms" "36" "98" "-1"
"17" "girl_jeans_05.gif" "bottoms" "tops n bottoms" "37" "98" "-1"
"18" "girl_trousers_06.gif" "bottoms" "tops n bottoms" "39" "98" "-1"
"19" "girl_jeans_06.gif" "bottoms" "tops n bottoms" "31" "97" "-1"
"20" "girl_trousers_03.gif" "bottoms" "tops n bottoms" "31" "97" "-1"
"21" "girl_trousers_09.gif" "bottoms" "tops n bottoms" "35" "98" "-1"
"22" "girl_hat_01.gif" "hats" "hats" "23" "18" "-1"
"23" "girl_jacket_01.gif" "tops" "jackets" "24" "66" "-1"
"24" "girl_jacket_02.gif" "tops" "jackets" "25" "66" "-1"
"25" "girl_jacket_03.gif" "tops" "jackets" "25" "67" "-1"
"26" "girl_jacket_04.gif" "tops" "jackets" "23" "64" "-1"
"27" "girl_coat_02.gif" "tops" "jackets" "23" "60" "-1"
"28" "girl_jacket_07.gif" "tops" "jackets" "25" "67" "-1"
"29" "girl_jacket_08.gif" "tops" "jackets" "23" "65" "-1"
"30" "girl_coat_03.gif" "tops" "jackets" "23" "64" "-1"
"31" "baby_01.gif" "extras" "infront" "17" "65" "-1"
"32" "baby_02.gif" "extras" "infront" "17" "65" "-1"
"33" "baby_03.gif" "extras" "infront" "17" "65" "-1"
"34" "cat_white.gif" "extras" "infront" "7" "108" "-1"
"35" "cat_brown.gif" "extras" "infront" "7" "108" "-1"
"36" "dog.gif" "extras" "infront" "3" "87" "-1"
"37" "girl_belt_01.gif" "extras" "infront" "39" "98" "-1"
"38" "girl_belt_02.gif" "extras" "infront" "42" "97" "-1"
"39" "girl_belt_03.gif" "extras" "infront" "42" "97" "-1"
"40" "girl_necklace_01.gif" "extras" "infront" "40" "69" "-1"
"41" "girl_necklace_02.gif" "extras" "infront" "43" "66" "-1"
"42" "girl_hairclip_02.gif" "extras" "infront" "42" "18" "-1"
"43" "girl_earrings_01.gif" "extras" "infront" "29" "51" "-1"
"44" "girl_earrings_02.gif" "extras" "infront" "31" "52" "-1"
"45" "girl_earrings_03.gif" "extras" "infront" "29" "53" "-1"
"46" "girl_hairclip_01.gif" "extras" "infront" "35" "15" "-1"
"47" "girl_makeup_01.gif" "extras" "infront" "37" "44" "-1"
"48" "girl_makeup_02.gif" "extras" "infront" "36" "43" "-1"
"49" "girl_makeup_03.gif" "extras" "infront" "37" "42" "-1"
"50" "girl_makeup_04.gif" "extras" "infront" "37" "42" "-1"
"51" "girl_makeup_05.gif" "extras" "infront" "37" "46" "-1"
"52" "girl_makeup_06.gif" "extras" "infront" "37" "46" "-1"
"53" "girl_scarf_01.gif" "extras" "infront" "36" "63" "-1"
"54" "girl_scarf_02.gif" "extras" "infront" "36" "63" "-1"
"55" "girl_scarf_03.gif" "extras" "infront" "26" "66" "-1"
"56" "girl_scarf_04.gif" "extras" "infront" "25" "64" "-1"
"57" "girl_scarf_05.gif" "extras" "infront" "29" "69" "-1"
"58" "girl_scarf_06.gif" "extras" "infront" "37" "65" "-1"
"59" "girl_scarf_07.gif" "extras" "infront" "36" "64" "-1"
"60" "girl_earmuffs_01.gif" "extras" "infront" "25" "26" "-1"
"61" "girl_hair_01.gif" "hair" "hair" "19" "21" "-1"
"62" "girl_hair_02.gif" "hair" "hair" "19" "21" "-1"
"63" "girl_hair_03.gif" "hair" "hair" "24" "21" "-1"
"64" "girl_hair_04.gif" "hair" "hair" "28" "25" "-1"
"65" "girl_hair_05.gif" "hair" "hair" "28" "25" "-1"
"66" "girl_hair_06.gif" "hair" "hair" "29" "19" "-1"
"67" "girl_hair_07.gif" "hair" "hair" "24" "9" "-1"
"68" "girl_hair_08.gif" "hair" "hair" "25" "26" "-1"
"69" "girl_hair_09.gif" "hair" "hair" "27" "24" "-1"
"70" "girl_hair_10.gif" "hair" "hair" "29" "26" "-1"
"71" "girl_hair_11.gif" "hair" "hair" "28" "26" "-1"
"72" "girl_hair_12.gif" "hair" "hair" "28" "26" "-1"
"73" "girl_hair_13.gif" "hair" "hair" "30" "27" "-1"
"74" "girl_hair_14.gif" "hair" "hair" "30" "26" "-1"
"75" "girl_hair_15.gif" "hair" "hair" "30" "25" "-1"
"76" "girl_hair_16.gif" "hair" "hair" "30" "17" "-1"
"77" "girl_hair_17.gif" "hair" "hair" "20" "27" "-1"
"78" "girl_hair_18.gif" "hair" "hair" "30" "16" "-1"
"79" "girl_hair_19.gif" "hair" "hair" "27" "23" "-1"
"80" "girl_hair_20.gif" "hair" "hair" "30" "22" "-1"
"81" "girl_hair_21.gif" "hair" "hair" "24" "23" "-1"
"82" "girl_hair_22.gif" "hair" "hair" "23" "23" "-1"
"83" "girl_hair_23.gif" "hair" "hair" "23" "24" "-1"
"84" "girl_hair_24.gif" "hair" "hair" "27" "25" "-1"
"85" "girl_hair_25.gif" "hair" "hair" "27" "18" "-1"
"86" "girl_hair_26.gif" "hair" "hair" "27" "21" "-1"
"87" "girl_hair_27.gif" "hair" "hair" "28" "19" "-1"
"88" "girl_hair_28.gif" "hair" "hair" "18" "16" "-1"
"89" "girl_hair_29.gif" "hair" "hair" "22" "7" "-1"
"90" "girl_hair_30.gif" "hair" "hair" "20" "25" "-1"
"91" "girl_hair_31.gif" "hair" "hair" "31" "22" "-1"
"92" "girl_hair_32.gif" "hair" "hair" "29" "22" "-1"
"93" "girl_hair_33.gif" "hair" "hair" "27" "23" "-1"
"94" "girl_hair_34.gif" "hair" "hair" "30" "25" "-1"
"95" "girl_hair_35.gif" "hair" "hair" "19" "23" "-1"
"96" "girl_undies_22.gif" "pants" "underware" "39" "70" "-1"
"97" "girl_undies_23.gif" "pants" "underware" "40" "70" "-1"
"98" "girl_undies_24.gif" "pants" "underware" "39" "71" "-1"
"99" "girl_undies_15.gif" "pants" "underware" "39" "71" "-1"
"100" "girl_undies_14.gif" "pants" "underware" "39" "70" "-1"
"101" "girl_undies_13.gif" "pants" "underware" "39" "70" "-1"
"102" "girl_undies_12.gif" "pants" "underware" "39" "70" "-1"
"103" "girl_undies_11.gif" "pants" "underware" "40" "82" "-1"
"104" "girl_undies_09.gif" "pants" "underware" "40" "72" "-1"
"105" "girl_undies_10.gif" "pants" "underware" "40" "77" "-1"
"106" "girl_hose_03.gif" "pants" "underware" "36" "105" "-1"
"107" "girl_hose_04.gif" "pants" "underware" "37" "108" "-1"
"108" "girl_hose_05.gif" "pants" "underware" "37" "107" "-1"
"109" "girl_hose_06.gif" "pants" "underware" "37" "107" "-1"
"110" "girl_hose_07.gif" "pants" "underware" "37" "109" "-1"
"111" "girl_hose_02.gif" "pants" "underware" "37" "107" "-1"
"112" "girl_undies_17.gif" "pants" "underware" "45" "96" "-1"
"113" "girl_undies_18.gif" "pants" "underware" "40" "80" "-1"
"114" "girl_undies_19.gif" "pants" "underware" "39" "70" "-1"
"115" "girl_undies_20.gif" "pants" "underware" "39" "69" "-1"
"116" "girl_undies_21.gif" "pants" "underware" "38" "68" "-1"
"117" "girl_shoes_05.gif" "shoes" "tops n bottoms" "36" "128" "-1"
"118" "girl_boots_02.gif" "shoes" "tops n bottoms" "36" "116" "-1"
"119" "girl_boots_03.gif" "shoes" "tops n bottoms" "37" "118" "-1"
"120" "girl_boots_04.gif" "shoes" "tops n bottoms" "37" "118" "-1"
"121" "girl_boots_07.gif" "shoes" "tops n bottoms" "35" "116" "-1"
"122" "girl_boots_05.gif" "shoes" "tops n bottoms" "35" "116" "-1"
"123" "girl_shoes_11.gif" "shoes" "tops n bottoms" "36" "122" "-1"
"124" "girl_shoes_12.gif" "shoes" "tops n bottoms" "36" "124" "-1"
"125" "girl_shoes_13.gif" "shoes" "tops n bottoms" "37" "125" "-1"
"126" "girl_boots_08.gif" "shoes" "tops n bottoms" "34" "118" "-1"
"127" "girl_boots_09.gif" "shoes" "tops n bottoms" "34" "118" "-1"
"128" "girl_shoes_06.gif" "shoes" "tops n bottoms" "36" "124" "-1"
"129" "girl_shoes_07.gif" "shoes" "tops n bottoms" "35" "130" "-1"
"130" "girl_shoes_08.gif" "shoes" "tops n bottoms" "35" "130" "-1"
"131" "girl_shoes_09.gif" "shoes" "tops n bottoms" "37" "131" "-1"
"132" "girl_shoes_10.gif" "shoes" "tops n bottoms" "36" "125" "-1"
"133" "girl_shoes_14.gif" "shoes" "tops n bottoms" "36" "125" "-1"
"134" "girl_boots_06.gif" "shoes" "tops n bottoms" "35" "116" "-1"
"135" "girl_hat_02.gif" "hats" "hats" "27" "19" "-1"
"138" "girl_hat_04.gif" "hats" "hats" "29" "19" "-1"
"137" "girl_hat_03.gif" "hats" "hats" "29" "18" "-1"
"139" "girl_top_02.gif" "tops" "tops n bottoms" "36" "68" "-1"
"140" "girl_wrap_02.gif" "extras" "infront" "28" "70" "-1"
"141" "girl_top_03.gif" "tops" "tops n bottoms" "40" "78" "-1"
"142" "girl_shirt_03.gif" "tops" "tops n bottoms" "27" "71" "-1"
"143" "girl_top_05.gif" "tops" "tops n bottoms" "39" "79" "-1"
"144" "girl_shirt_04.gif" "tops" "tops n bottoms" "26" "67" "-1"
"145" "girl_top_07.gif" "tops" "tops n bottoms" "39" "69" "-1"
"146" "girl_top_08.gif" "tops" "tops n bottoms" "38" "70" "-1"
"147" "girl_tee_01.gif" "tops" "tops n bottoms" "34" "70" "-1"
"148" "girl_tee_02.gif" "tops" "tops n bottoms" "33" "68" "-1"
"149" "girl_tee_03.gif" "tops" "tops n bottoms" "33" "69" "-1"
"150" "girl_tee_04.gif" "tops" "tops n bottoms" "33" "69" "-1"
"151" "girl_tee_05.gif" "tops" "tops n bottoms" "33" "69" "-1"
"152" "girl_top_10.gif" "tops" "tops n bottoms" "27" "69" "-1"
"153" "girl_top_04.gif" "tops" "tops n bottoms" "39" "70" "-1"
"154" "girl_top_06.gif" "tops" "tops n bottoms" "39" "70" "-1"
"155" "girl_shirt_05.gif" "tops" "tops n bottoms" "27" "67" "-1"
"156" "girl_top_09.gif" "tops" "tops n bottoms" "39" "70" "-1"
"157" "girl_top_01.gif" "tops" "tops n bottoms" "40" "70" "-1"
"158" "girl_jumper_07.gif" "tops" "tops n bottoms" "27" "66" "-1"
"159" "girl_jumper_08.gif" "tops" "tops n bottoms" "27" "66" "-1"
"160" "girl_dress_08.gif" "tops" "tops n bottoms" "30" "67" "-1"
"161" "girl_dress_09.gif" "tops" "tops n bottoms" "36" "66" "-1"
"162" "girl_dress_10.gif" "tops" "tops n bottoms" "37" "69" "-1"
"163" "girl_dress_11.gif" "tops" "tops n bottoms" "31" "69" "-1"
"164" "girl_dress_12.gif" "tops" "tops n bottoms" "35" "69" "-1"
"165" "girl_dress_07.gif" "tops" "tops n bottoms" "37" "66" "-1"
"190" "shades_01.gif" "extras" "infront" "34" "46" "-1"
"166" "cat_black.gif" "extras" "infront" "7" "108" "-1"
"168" "girl_hair_36.gif" "hair" "hair" "18" "20" "-1"
"167" "girl_hat_05.gif" "hats" "hats" "20" "11" "-1"
"169" "girl_hair_37.gif" "hair" "hair" "25" "24" "-1"
"170" "girl_hair_38.gif" "hair" "hair" "21" "24" "-1"
"171" "girl_hair_39.gif" "hair" "hair" "27" "24" "-1"
"172" "girl_hair_40.gif" "hair" "hair" "24" "26" "-1"
"173" "girl_hair_41.gif" "hair" "hair" "27" "22" "-1"
"174" "girl_hair_42.gif" "hair" "hair" "29" "19" "-1"
"175" "girl_hair_43.gif" "hair" "hair" "26" "22" "-1"
"176" "girl_hair_44.gif" "hair" "hair" "26" "24" "-1"
"177" "girl_hair_45.gif" "hair" "hair" "20" "9" "-1"
"178" "girl_hair_46.gif" "hair" "hair" "14" "23" "-1"
"179" "girl_hair_47.gif" "hair" "hair" "25" "22" "-1"
"180" "girl_hair_48.gif" "hair" "hair" "24" "23" "-1"
"181" "girl_hair_49.gif" "hair" "hair" "25" "22" "-1"
"182" "girl_hair_50.gif" "hair" "hair" "28" "24" "-1"
"183" "girl_hair_51.gif" "hair" "hair" "28" "23" "-1"
"184" "girl_hair_52.gif" "hair" "hair" "30" "19" "-1"
"185" "girl_hair_53.gif" "hair" "hair" "30" "19" "-1"
"186" "girl_hair_54.gif" "hair" "hair" "27" "25" "-1"
"187" "girl_hair_55.gif" "hair" "hair" "25" "21" "-1"
"188" "girl_hair_56.gif" "hair" "hair" "29" "22" "-1"
"189" "girl_jumper_06.gif" "tops" "tops n bottoms" "27" "70" "-1"
"191" "shades_02.gif" "extras" "infront" "36" "47" "-1"
"192" "shades_03.gif" "extras" "infront" "35" "46" "-1"
"193" "shades_04.gif" "extras" "infront" "36" "46" "-1"
"194" "girl_bag_05.gif" "extras" "infront" "57" "77" "-1"
"195" "girl_bag_06.gif" "extras" "infront" "20" "83" "-1"
"196" "girl_bag_07.gif" "extras" "infront" "26" "71" "-1"
"197" "girl_bag_08.gif" "extras" "infront" "14" "86" "-1"
"198" "computer.gif" "extras" "infront" "2" "100" "-1"
"199" "glasses_01.gif" "extras" "infront" "35" "44" "-1"
"200" "glasses_02.gif" "extras" "infront" "34" "47" "-1"
"201" "glasses_03.gif" "extras" "infront" "36" "48" "-1"
"202" "glasses_04.gif" "extras" "infront" "36" "48" "-1"
"203" "glasses_05.gif" "extras" "infront" "36" "47" "-1"
"204" "glasses_06.gif" "extras" "infront" "36" "47" "-1"
"205" "glasses_07.gif" "extras" "infront" "36" "47" "-1"
"206" "girl_skirt_13.gif" "bottoms" "tops n bottoms" "35" "95" "-1"
"207" "girl_skirt_14.gif" "bottoms" "tops n bottoms" "35" "96" "-1"
"208" "girl_eyes_blue.gif" "extras" "infront" "36" "46" "-1"
"209" "girl_eyes_brown.gif" "extras" "infront" "36" "45" "-1"
"210" "girl_eyes_green.gif" "extras" "infront" "36" "45" "-1"
"211" "bindi.gif" "extras" "infront" "47" "41" "-1"
"212" "girl_headscarf_01.gif" "hats" "hats" "28" "23" "-1"
"213" "girl_headscarf_02.gif" "hats" "hats" "28" "24" "-1" """

def getDataSet():
    set = []
    items = configdata.split('\n')
    for item in items:
        item = item.split(' ')
        set.append(map(lambda s:s.strip('"'), item))
    return set
    
    
def wget (fname):
    os.spawnlp(os.P_WAIT, '/sw/bin/wget', 'wget', fname)
    
path=r'http://www.stortroopers.com/fashion_girl/data/images/articles/'
elements = getDataSet()
for e in elements:
    print e[ImageName]
    wget(path + e[ImageName])

